apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -398,9 +409,15 @@ export function SubscriberOverviewForm(props: SubscriberOverviewFormProps) {
         open={isDeleteModalOpen}
         onOpenChange={setIsDeleteModalOpen}
         onConfirm={async () => {
+          const isLastSubscriber = data?.data.length === 1;
           await deleteSubscriber({ subscriberId: subscriber.subscriberId });
           setIsDeleteModalOpen(false);
-          navigate('../', { relative: 'path' });
+
+          if (isLastSubscriber) {
+            navigateToSubscribersFirstPage();
+          } else {
+            navigateToSubscribersCurrentPage();
+          }
```

Comment:
When deleting the last subscriber --> move to the subscriber's first page, otherwise to the current